### PR TITLE
fix: Adds missing reference to the endpoint_url for the S3 proxy

### PR DESCRIPTION
**Pull request by Nastaliss**
The endpoint_url reference in the method to generate file download URL is missing. This PR fixes that

### CODE REVIEW

**Comment by frgfm** [comment]
@Nastaliss Would you mind adding a PR description and creating an issue referencing all the PRs on that topic please? :pray:

File: src/app/services/bucket/s3.py [hunk 1]
```diff
@@ -31,6 +31,7 @@ def __init__(
         self, region: str, endpoint_url: str, access_key: str, secret_key: str, bucket_name: str, proxy_url: str
     ) -> None:
         _session = boto3.Session(access_key, secret_key, region_name=region)
+        self.endpoint_url = endpoint_url
```
Comment (frgfm) [inline src/app/services/bucket/s3.py:34]: No need we have 
```python
self._s3.meta.endpoint_url
```

**Review by frgfm — CHANGES_REQUESTED** [review]
No need for an extra instance attribute, I'll edit the PR :+1:

**Comment by Nastaliss** [comment]
[issue](https://github.com/pyronear/pyro-api/issues/300)

**Pull request merged**